Stream_support/include/CGAL/IO/WKT.h

```diff
@@ -0,0 +1,395 @@
+// Copyright (c) 2018  GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is part of CGAL (www.cgal.org); you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation; either version 3 of the License,
+// or (at your option) any later version.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0+
+//
+// Author(s)     : Maxime Gimeno
+
+#ifndef CGAL_WKT_H
+#define CGAL_WKT_H
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include <boost/geometry.hpp>
```

Comment:
Isn't `boost/geometry/io/wkt/wkt.hpp` sufficient?

Reply:
Not according to the doc of boost, but it seems you are wright.

Reply:
In the [documentation of `read_wtk`](https://www.boost.org/doc/libs/1_65_1/libs/geometry/doc/html/geometry/reference/io/wkt/read_wkt.html), for example, I read:
> Header
> 
> Either
> 
>     #include <boost/geometry.hpp>
> 
> Or
> 
>     #include <boost/geometry/io/wkt/read.hpp>

Reply:
None of which is boost/geometry/io/wkt/wkt.hpp.

Reply:
Then use `<boost/geometry/io/wkt/read.hpp>` or `<boost/geometry/io/wkt/write.hpp>`, or both, depending on what is exactly required by the CGAL headers.